### DISCUSSION TITLE
export data from CARLA in KITTI dataset format

### DISCUSSION THREAD

**Discussion by husamhamu** [carla-simulator/carla#5019] [Q&A]
HI, I need to collect detection and tracking data for monocular 3d object detection in KITTI format.

so I wonder if anyone has worked on generating data from the new versions of CARLA( version 0.9x) in KITTI format. There is a previous work on this repo: [https://github.com/Ozzyz/carla-data-export?utm_source=pocket_mylist](url) that is compatible with CARLA_0.8.4, but I tried the code with CARLA_0.9.13 and didn't seem to work?

Anyone can help please?

**Comment by husamhamu** [comment]
So far I saw this pull request [https://github.com/carla-simulator/carla/pull/4992](url) and this file: [https://github.com/carla-simulator/carla/blob/master/PythonAPI/examples/client_bounding_boxes.py](url) that creates 3d bounding boxes that are not quite synchronous
but I still don't know how to make use out of these. 

Is it like no one is interested in the topic ?

**Reply by RaviBeagle** [reply]
This is a more recent effort to do the same: https://github.com/jedeschaud/kitti_carla_simulator
And I am able to run it CARLA 0.9.12 on Linux without any issues.
